components/pipeline-service/staging/base/main-pipeline-service-configuration.yaml

```diff
@@ -1781,6 +1781,136 @@ spec:
                       requests:
                         cpu: 400m
                         memory: 1Gi
+        pipelines-as-code-controller:
+          spec:
+            template:
+              spec:
+                containers:
+                  - name: pac-controller
+                    resources:
+                      limits:
+                        cpu: 400m
+                        memory: 2.5Gi
+                      requests:
+                        cpu: 400m
+                        memory: 2.5Gi
+        pipelines-as-code-watcher:
+          spec:
+            template:
+              spec:
+                containers:
+                  - name: pac-watcher
+                    resources:
+                      limits:
+                        cpu: 500m
+                        memory: 1Gi
+                      requests:
+                        cpu: 500m
+                        memory: 1Gi
+        pipelines-as-code-webhook:
+          spec:
+            template:
+              spec:
+                containers:
+                  - name: pac-webhook
+                    resources:
+                      limits:
+                        cpu: 400m
+                        memory: 100Mi
+                      requests:
+                        cpu: 400m
+                        memory: 100Mi
+        pipelines-console-plugin:
+          spec:
+            template:
+              spec:
+                containers:
+                  - name: pipelines-console-plugin
+                    resources:
+                      limits:
+                        cpu: 200m
+                        memory: 8Mi
+                      requests:
+                        cpu: 200m
+                        memory: 8Mi
+        tekton-chains-controller:
+          spec:
+            template:
+              spec:
+                containers:
+                  - name: tekton-chains-controller
+                    resources:
+                      limits:
+                        cpu: 500m
```

Comment:
this seems low. I see some peaks on one of the clusters going above 800m, so probably 1000m will be a better value.
And in general, it feels most value a bit conservative and we should be able to withstand peaks x2, x3 or even x5 above the normal load.

Reply:
Updated. My reasoning here was generally ~1.5-2x highest spike I saw, rounded to a friendly number. I must have missed this when looking at CPU though. I've revised the numbers now